lib/tfa/storage.rb

```diff
@@ -2,8 +2,8 @@ module TFA
   class Storage
     include Enumerable
 
-    def initialize(filename:)
-      @storage = PStore.new(File.join(Dir.home, ".#{filename}.pstore"))
+    def initialize(options)
```

Comment:
This gem was targeting ruby 2+ syntax. Hence, the default parameter. Switching to a hash is cool.